.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,8 +2,13 @@
 guidelines to help us review it. You can remove these comments
 as you fill out this template. -->
 
+<!--DO NOT edit files and directories listed in _NOT_EDITED_HERE.txt
+as these are maintained in upstream repos and added to the central
+docs repo periodically. Your PR will fail automated checks if it
+includes changes to these files or directories.-->
+
 <!-- **Squash your commits**: Please use an
-[interactive rebase](https://help.github.com/articles/about-git-rebase/)
+interactive rebase (https://help.github.com/articles/about-git-rebase/)
```

Comment:
oh, not related, but I think we explain those steps in our contributors guide as well https://docs.docker.com/opensource/workflow/work-issue/#/pull-and-rebase-frequently